### PR TITLE
[Konvoy 2.1.1] Release Notes

### PR DESCRIPTION
**Pull request by mwrightd2**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-82312?filter=-1

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by cmastr** [comment]
this will need to be moved to the feature/release-2.1.1 branch

File: pages/dkp/konvoy/2.1/release-notes/2.1.0/index.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+For more information on using Nvidia with Konvoy, see the [GPU](../choose-infrastructure/aws/gpu) documentation.
+
+#### Easier Air-gapped deployments
+
+Konvoy 2.1 comes with an easier way to deploy in an air gapped environment. Rather than one large package that includes the requirements for all air gapped environments, you can now create a smaller package that is specific to your requirements and can fit on a laptop or USB.
+
+#### DKP Licensing through Amazon Marketplace
+
+You can purchase a license for Konvoy or Kommander through the AWS Marketplace, and then add it to Kommander. In the Kommander UI, you can see information such as the license status (valid or expired), the license capacity (number of cores or clusters), and expiration date.
+
```
Comment (Colin1841) [inline pages/dkp/konvoy/2.1/release-notes/2.1.0/index.md:62]:
```suggestion

### Fixes and Improvements

#### DKP delete fails in AWS

Fixed an issue where the 'dkp delete' command could fail with a SIGSEGV when attempting to delete a DKP cluster from AWS where you have permanent credentials. (COPS-7109)

#### DKP installer fails to follow respective AWS --region or Azure --location flags

Fixed an issue where the AWS `--region` or Azure `--location` installer flags were not being enforced in the target cluster. (COPS-7101)

#### DKP move command not moving some items

We corrected an issue where the `PreprovisionedInventory` object and SSH key secret were not moved to the target cluster when making the cluster self-managing.(COPS-7079)

```

File: pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+### New features and capabilities
+
+#### DKP Major Version Upgrade
+
+Konvoy and Kommander 2.1 represent a major version upgrade that moves forward DKP architecture to give you access to D2iQ's next generation centralized Kubernetes and smart cloud native applications. It incorporates ClusterAPI as a major re-architecture in its management of production Kubernetes clusters. [ClusterAPI](https://cluster-api.sigs.k8s.io/introduction.html), or CAPI, enables declarative creation, configuration, and management of clusters. Declarative mode is a Kubernetes best practice that simplifies lifecycle tasks, making them more consistent and more repeatable. 2.1 enhances your existing clusters to use a new architecture.
+
+#### Other Feature Additions
+
+* Provision a cluster using an advanced YAML editor.
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md:39]:
```suggestion
* Added the ability to provision a cluster using an advanced YAML editor.
```

File: pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+* More bootstrap images are included with air-gapped bundles that contain OS packages.
```
Comment (cneth) [inline pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md:41]: I don't think this sentence makes much sense as is.   I'm happy to try and reword it if someone can point me at the JIRA ticket for the work that was done here.

**Review by cneth — APPROVED** [review]

File: pages/dkp/konvoy/2.1/release-notes/2.1.0/index.md (hunk 1 above)
Comment (cmastr) [reply pages/dkp/konvoy/2.1/release-notes/2.1.0/index.md:62]: agree

File: pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md (hunk 3 above)
Comment (mwrightd2) [reply pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md:41]: @cneth Check your Slack DMs, ive reached out to you re: this.

File: pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes Konvoy 2.1.1
+title: Release Notes Konvoy 2.1.1
+menuWeight: 10
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md:5]:
```suggestion
menuWeight: 20
```

**Review by cmastr — APPROVED** [review]

File: pages/dkp/konvoy/2.1/release-notes/2.1.0/index.md (hunk 1 above)
Comment (mwrightd2) [reply pages/dkp/konvoy/2.1/release-notes/2.1.0/index.md:62]: Not adding these commits, putting them into their appropriate place in 2.1.1 index.md.

**Review by mwrightd2 — COMMENTED** [review]
Reviewed!

File: pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+#### DKP installer fails to follow respective AWS --region or Azure --location flags
+
+Fixed an issue where the AWS `--region` or Azure `--location` installer flags were not being enforced in the target cluster. (COPS-7101)
+
+#### DKP move command not moving some items
+
+We corrected an issue where the `PreprovisionedInventory` object and SSH key secret were not moved to the target cluster when making the cluster self-managing.(COPS-7079)
+
+### Component updates
```
Comment (cmastr) [inline pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md:59]: Are these the right versions?

File: pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+The following components have been upgraded to the listed version:
+
+- Calico 3.20
+- AWS EBS CSI 1.4
+- CSI External Snapshotter 4.2.1
+- Azure CSI 1.8.0
+- Local Static Provisioner CSI 2.4.0
+- Cluster Autoscaler 1.21.0
+- Node Feature Discovery 0.8.2
+- Nvidia Node Feature Discovery 0.4.1
```
Comment (dkoshkin) [inline pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md:70]:
```suggestion
- Cluster API Preprovisioned Provider v0.4.0
- Konvoy Image Builder v1.5.0
```

File: pages/dkp/konvoy/2.1/release-notes/2.1.0/index.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+For more information on creating and sending a Konvoy Diagnostics Bundle, see [Generate a Support Bundle](../../troubleshooting/generate-a-support-bundle).
+
+#### Support for NVIDIA 470 Driver
+
+Konvoy 2.1 supports the upgraded Nvidia 470.x driver and DCGM exporter 2.2.9, which enables technology for running DKP on NVIDIA DGX. NVIDIA DGX customers now have a robust Kubernetes platform in Konvoy and Kommander that vastly simplifies the process of getting the infrastructure needed for applications up and running on this powerful hardware platform.
+
+For more information on using Nvidia with Konvoy, see the [GPU](../../choose-infrastructure/aws/gpu) documentation.
+
+#### Easier Air-gapped deployments
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.1/release-notes/2.1.0/index.md:55]: I think this is lower case?

```suggestion
#### Easier air-gapped deployments
```

File: pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+#### Other Feature Additions
+
+* Added the ability to provision a cluster using an advanced YAML editor.
+
+### Fixes and Improvements
+
+#### DKP delete fails in AWS
+
+Fixed an issue where the 'dkp delete' command could fail with a SIGSEGV when attempting to delete a DKP cluster from AWS where you have permanent credentials. (COPS-7109)
```
Comment (msdolbey) [inline pages/dkp/konvoy/2.1/release-notes/2.1.1/index.md:49]: Making this code style - just pushing

```suggestion
Fixed an issue where the `dkp delete` command could fail with a SIGSEGV when attempting to delete a DKP cluster from AWS where you have permanent credentials. (COPS-7109)
```

**Comment by msdolbey** [comment]
@mwrightd2 @cmastr - there I changed the structure (but not the content) here - made the `/release-notes` page the landing page, so deleted the content that is now in `/release-notes/2.1.0` - so you'll need to pull before you make any more changes.

**Comment by msdolbey** [comment]
Okay - that was a nasty merge conflict - but should be all set now

FYI @mwrightd2 @cmastr

**Review by cmastr — APPROVED** [review]

**Pull request merged**